pipeline/lookup.csv

```diff
@@ -29678,3 +29678,892 @@ brownfield-land,,local-authority-eng:BAE,BFR53,1729669
+brownfield-land,,local-authority-eng:BRD,SH/061,1729821
+brownfield-land,,local-authority-eng:BRD,SH/062,1729822
+brownfield-land,,local-authority-eng:BRD,SH/063,1729823
+brownfield-land,,local-authority-eng:BRD,SH/064,1729824
+brownfield-land,,local-authority-eng:BRD,SH/065,1729825
+brownfield-land,,local-authority-eng:BRD,SH/067A,1729826
+brownfield-land,,local-authority-eng:BRD,SH/067B,1729827
+brownfield-land,,local-authority-eng:BRD,SH/068,1729828
+brownfield-land,,local-authority-eng:BRD,SH/074,1729829
+brownfield-land,,local-authority-eng:BRD,SH/084,1729830
+brownfield-land,,local-authority-eng:BRD,SI/024,1729831
+brownfield-land,,local-authority-eng:BRD,SW/035,1729832
+brownfield-land,,local-authority-eng:BRD,SW/088,1729833
+brownfield-land,,local-authority-eng:BRD,SW/157,1729834
+brownfield-land,,local-authority-eng:BRD,SW/158,1729835
+brownfield-land,,local-authority-eng:BRD,SW/160,1729836
+brownfield-land,,local-authority-eng:BRD,SW/163,1729837
+brownfield-land,,local-authority-eng:BRD,SW/165,1729838
+brownfield-land,,local-authority-eng:BRD,SW/168,1729839
+brownfield-land,,local-authority-eng:BRD,SW/170,1729840
+brownfield-land,,local-authority-eng:BRD,TH/037,1729841
+brownfield-land,,local-authority-eng:BRD,WI/002,1729842
+brownfield-land,,local-authority-eng:BRC,Map Ref 12 BRA7 Eastern Gateway Development Area, Bracknell (Town Square) (Submission Local Plan),1729843
+brownfield-land,,local-authority-eng:BRC,Map Ref 14 BRA12 Former Bus Depot (Submission Local Plan),1729844
+brownfield-land,,local-authority-eng:BRC,Map Ref 43 BRA 17 Bus Station (part of the Southern Gateway Development Area, Bracknell) (Submission Local Plan),1729845
+brownfield-land,,local-authority-eng:BRC,Map Ref 55,1729846
+brownfield-land,,local-authority-eng:BRC,Map ref 57 LP8 BRA18 The Peel Centre and The Point (Submission Local Plan),1729847
+brownfield-land,,local-authority-eng:BRC,Map ref 58,1729848
+brownfield-land,,local-authority-eng:BRC,Map ref 59,1729849
+brownfield-land,,local-authority-eng:BRC,Map ref 60,1729850
+brownfield-land,,local-authority-eng:BRC,Map ref 61,1729851
+brownfield-land,,,Map ref 62,1729852
+brownfield-land,,,Map ref 63,1729853
+brownfield-land,,local-authority-eng:CAT,BR-119,1729854
+brownfield-land,,local-authority-eng:CAT,BR-121,1729855
+brownfield-land,,local-authority-eng:CAT,BR-122,1729856
+brownfield-land,,local-authority-eng:CAT,BR-123,1729857
+brownfield-land,,local-authority-eng:CAT,BR-124,1729858
+brownfield-land,,local-authority-eng:CAT,BR-125,1729859
+brownfield-land,,local-authority-eng:CAT,BR-127,1729860
+brownfield-land,,local-authority-eng:CAT,BR-128,1729861
+brownfield-land,,local-authority-eng:CAT,BR-129,1729862
+brownfield-land,,local-authority-eng:CAT,BR-117,1729863
+brownfield-land,,local-authority-eng:CAT,BR-118,1729864
+brownfield-land,,local-authority-eng:CAT,BR-120,1729865
+brownfield-land,,local-authority-eng:CAT,BR-126,1729866
+brownfield-land,,local-authority-eng:CAR,,1729867

```

Comment:
This and rows below have no reference this shouldn't be in the lookup and it's weird that it's happening.